### PR TITLE
Displaying of success message on loading watchers pane.

### PR DESCRIPTION
**Pull request by knight-bubble**
[`* `#19219` [DETAILS PANE] SUCCESS MESSAGE "WATCHER SUCCESSFULLY ADDED" WRONGLY DISPLAYED`](http://community.openproject.org/work_packages/19219)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-19219-watcher-success-message.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-19219-watcher-success-message.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/tests/unit/tests/work_packages/controllers/details-tab-watchers-controller-test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      I18n: $injector.get('I18n'),
+      ADD_WATCHER_SELECT_INDEX: $injector.get('ADD_WATCHER_SELECT_INDEX')
+    });
+  }));
+
+  describe('addWatcherSuccess', function() {
+    it('is not called on initialization', function() {
+      sinon.spy(scope, 'outputMessage');
+      expect(
+        scope
```
Comment (myabc) [inline frontend/tests/unit/tests/work_packages/controllers/details-tab-watchers-controller-test.js]: @hirtie-maxim you can use the [sinon-chai assertions](https://github.com/domenic/sinon-chai#assertions).. e.g. `expect(scope.outputMessage).to.been.calledWith(...`

**Comment by myabc** [comment]
@hirtie-maxim looks fine code-wise (just one small suggestion).

**Comment by floriank** [comment]
LGTM, mind the possible improvement as noted by @myabc

**Pull request merged**